config/config.go

```diff
@@ -12,10 +12,15 @@ type Config struct {
 	GracefulShutdownTimeout    time.Duration `envconfig:"GRACEFUL_SHUTDOWN_TIMEOUT"`
 	HealthCheckInterval        time.Duration `envconfig:"HEALTHCHECK_INTERVAL"`
 	HealthCheckCriticalTimeout time.Duration `envconfig:"HEALTHCHECK_CRITICAL_TIMEOUT"`
-	KafkaAddr                  []string      `envconfig:"KAFKA_ADDR"                     json:"-"`
+	KafkaAddr                  []string      `envconfig:"KAFKA_ADDR"                   json:"-"`
```

Comment:
```suggestion
	KafkaAddr                  []string      `envconfig:"KAFKA_ADDR"`
```
no need to hide this value